pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-  "setuptools >= 64.0.0",  # minimum for PEP 660 editable install
+  "setuptools == 77.0.1",  # minimum for PEP 660 editable installs
```

Comment:
Did you still mean to bump the minimum required version?

Reply:
No, that's an error in a rebase. Will fix. Thanks for catching and sorry that this sloppily slipped through. Pushed fix.

Reply:
No probs.